### ISSUE TITLE
Revision: 211 Bookmobiles / Addition: Number of Outreach Vehicles

### ISSUE THREAD

**Issue by KristenCooke** [IMLS/public-libraries-survey#72]
Kristen Cooke

State/Affiliation:
Arkansas State Library 

Description of Change:
Revise 211 Number of Bookmobiles 

Current Definition:
211	Number of Bookmobiles
A bookmobile is a traveling branch library. It consists of at least all the following:
•	A truck or van that carries an organized collection of library materials; 
•	Paid staff; and 
•	Regularly scheduled hours (bookmobile stops) for being open to the public. 

Note: Count the number of vehicles in use, not the number of stops the vehicle makes.

Revised: 
211	Number of Outreach Vehicles
211a	How many of the reported outreach vehicles are bookmobiles that are operated by paid staff, have regularly scheduled hours, and carry an organized collection of library materials?

Revised Definition:
211 	Number of Outreach Vehicles
An outreach vehicle is a vehicle designed and equipped to offer offsite library services. Outreach vehicles may or may not have standard scheduled routes or carry an organized collection. Do not include vehicles that are solely utilized for the purpose of transporting staff or volunteers to and from outreach events. Do not include courier vehicles that are specifically utilized for the transport of materials between outlets. Include vehicles equipped for mobile job labs, pop-up libraries, mobile programming, 

Note: Count the number of vehicles in use, not the number of stops the vehicle makes.

211a	Bookmobiles
Of the number of outreach vehicles reported, report the number of vehicles that consists of the following:  
•	A truck or van that carries an organized collection of library materials; 
•	Paid staff; and 
•	Regularly scheduled hours (bookmobile stops) for being open to the public.

To be counted as a Bookmobile, the vehicle must meet all three of the outlined characteristics.
Vehicles not meeting all three of the outlined characteristics should be reported in 211 Number of Outreach Vehicles. 

Note: Count the number of vehicles in use, not the number of stops the vehicle makes.
Justification:
The current measurement of bookmobile leaves out vehicles that are exclusively utilized for outreach and community programming but may not meet the standards of a regular scheduled stop or have organized collections of materials. As mobile outreach modernizes more vehicles are designed for fluid use and multipurpose with collections being temporary or curated for the specific outreach event. The current element accurately accounts for traditional bookmobile services but does not account for modern outreach vehicles that do not have a core collection and standard route, but still provide core library services via vehicle. As mobile outreach evolves, it becomes important to measure both service-typical bookmobile access and tell the story of what is happening with modern mobile outreach. 

Measuring the total number of outreach vehicles while also isolating the number of traditional bookmobiles allows for the count of both types of access services. Isolating traditional bookmobiles as a sub-count of outreach vehicles prevents a disruption in historical data measurement of bookmobiles while allowing for the addition of a new measure of outreach vehicles. 

Measuring these vehicles will also support the work of SLAAs that support e-rate applications since outreach vehicles may meet the standards for e-rate but not meet the PLS definition of a bookmobile leaving the SLAA without an accurate count of library vehicles qualified for federal programs.

Potential methodological issues:
This method should preserve and allow for the isolation and continued measurement of data on traditional bookmobile service and allow for the tracking of outreach vehicles that do not meet the previous definition. In implementation, SDCs may experience an increased training load to ensure data submitters understand the distinction and the bookmobile counts remain accurate. 

Other considerations would be whether the measurement of Outreach Vehicles would necessitate an FSCS ID and then be counted as an outlet.

**Comment by KristenCooke** [comment]
In reviewing Evan's proposal, the suggestion could be amended to add an element that measures the total number of outreach vehicles including bookmobile reported.

**Comment by klow-air** [comment]
I'll add to this since Kristen got to this before I did. The **addition of a measure for outreach vehicles** was suggested during the Summer LSWG meeting, but it was not tested during the FY 2025 period and is on the docket for FY 2026.

**Comment by megdepriest** [comment]
This is a great addition! Preserving the Bookmobile data AND adding other mobile outreach vehicles would be great. If this goes forward, I think it will be helpful to define "vehicle" We will receive questions seeking clarification for non-motorized, mobile outreach such as bikes. paleta carts, trailers, etc.

**Comment by enielsen-air** [comment]
This proposal is related to #67 in that data element 211 is a count of the number of bookmobile records on the Outlet file. So in order to make "outreach vehicles" the overall category and "bookmobiles" a subset of outreach vehicles, we have to change the definition of an Outlet to include vehicles (and maybe facilities) that do not have a regular collection or schedule of open hours.

**Comment by megdepriest** [comment]
Makes sense we should not make bookmobiles a subset of outreach vehicles if we don't want to mess with the definition of outlets. What if it were a stand-alone question? 

Number of Outreach Vehicles that are not Bookmobiles. An outreach vehicle is a vehicle designed and equipped to offer offsite library services. Unlike Bookmobiles, outreach vehicles may not have standard scheduled routes or carry an organized collection. Do not include Bookmobiles. Do not include vehicles that are solely used for the purpose of transporting staff or volunteers to and from outreach events. Do not include courier vehicles that are specifically utilized for the transport of materials between outlets. Include vehicles with mobile computer labs, pop-up libraries, mobile programming, and informational materials and services. Vehicles include vans, trucks, scooters, bikes, carts, and trailers. 

Note: Count the number of vehicles in use, not the number of stops the vehicle makes.

....or something like that

**Comment by KristenCooke** [comment]
@megdepriest a standalone question makes sense. The distinction between bookmobile and OVs remains clear and the specific addition of scooters, bikes carts and trailers is something I did not think of, but am sure would be highly relevant in several states.

**Comment by JHoganHI** [comment]
I highly support the addition of outreach vehicles. Rural libraries in HI are getting more creative at adding additional points of access, especially after losing an outlet in Lahaina. Thank you Kristen and LSWG members for continuing this proposal.

**Comment by megdepriest** [comment]
I just had to share these brand new electronic Book Bikes the San Diego Public Library just launched into the world. Just because!

![Open San Diego Book Bike](https://github.com/user-attachments/assets/071d6fd1-3431-4722-82d3-d6fb40bfea22)
![San Diego Book Bikes](https://github.com/user-attachments/assets/5c423c5c-4cb8-4951-a211-cebc238caeeb)

**Comment by nrgieske** [comment]
Kentucky I believe has more bookmobiles on the road than any other state (74 currently) so I would not be in favor of deleting the bookmobile element. Kentucky has also collected information on Outreach vehicles for years and I often forget it is not a federal element and have wondered why it is not. We currently have 89 outreach vehicles. I am in favor of keeping bookmobiles and adding Outreach vehicles as a new element, I would be happy to share how Kentucky's outreach section is set-up and the questions asked.
